lib/ansible/plugins/lookup/csvfile.py

```diff
@@ -142,8 +150,9 @@ def run(self, terms, variables=None, **kwargs):
 
             # parameters specified?
             try:
-                for param in params[1:]:
-                    name, value = param.split('=')
+                for name, value in iteritems(kv):
```

Comment:
For this use case, I think `kv.items()` would be fine rather than having to import `iteritems`.
```suggestion
                for name, value in kv.items():
```

Reply:
Fixed and also stopped using `unquote()` that could break people and added a note that for historical reasons, the search key must be unquoted.